packaging/nuget/Particular.CodeRules.nuspec

```diff
@@ -2,8 +2,10 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/10/nuspec.xsd">
     <metadata>
         <id>Particular.CodeRules</id>
-        <version>$version$</version>
+        <version>$version$-beta</version>
```

Comment:
It allows the package to compile. Without the beta this package won't be generated since it has a dependency on StyleCop.Analyzers which is a beta package.